firebase-dataconnect/CHANGELOG.md

```diff
@@ -23,6 +23,13 @@
 * [changed] Replaced java.util.Date with
   com.google.firebase.dataconnect.LocalDate.
   ([#6434](https://github.com/firebase/firebase-android-sdk/pull/6434))
+* [changed] `DateSerializer` removed, as it is superceded by
+  `LocalDateSerializer`.
+  As of Data Connect emulator version 1.7.0, the generated Kotlin code uses
+  `com.google.firebase.dataconnect.LocalDate` instead of `java.util.Date`.
+  Therefore, this version of the SDK must be paired with an appropriate version
+  of the Data Connect emulator.
+  ([#6513](https://github.com/firebase/firebase-android-sdk/pull/6513))
 
```

Comment:
For devs, do they just need to regenerate code to use the new serializer?

Reply:
Yes, that is correct. I've tweaked the change log entry to read as follows, which hopefully clears up your question:

> `DateSerializer` removed, as it is superceded by `LocalDateSerializer`. As of Data Connect emulator version 1.7.0, the generated Kotlin code uses `com.google.firebase.dataconnect.LocalDate` instead of `java.util.Date`. Therefore, this version of the SDK must be paired with code generated by version 1.7.0 (or later) of the Data Connect emulator.